Cargo.toml

```diff
@@ -214,7 +214,7 @@ subxt-signer = { version = "0.34", default-features = false }
 celestia-types = "0.6.1"
 bech32 = "0.11.0"
 ripemd = "0.1.3"
-tonic = { version = "0.11.0", default-features = false }
+tonic = { version = "0.11.0", default-features = false, features=["tls","tls-roots","transport"] }
```

Comment:
it is enabled in the crate with the implementations of DA clients: https://github.com/matter-labs/zksync-era/blob/main/core/node/da_clients/Cargo.toml#L53